cmake/external_nnp_transformer.cmake

```diff
@@ -33,7 +33,7 @@ if (NGRAPH_NNP_ENABLE)
     if (DEFINED CUSTOM_NNP_TRANSFORMER_GIT_REPOSITORY)
         set(NNP_TRANSFORMER_GIT_REPOSITORY ${CUSTOM_NNP_TRANSFORMER_GIT_REPOSITORY})
     else()
-        set(NNP_TRANSFORMER_GIT_REPOSITORY git@github.com:NervanaSystems/argon-transformer.git)
+        set(NNP_TRANSFORMER_GIT_REPOSITORY https://github.com/NervanaSystems/nnp-transformer.git)
```

Comment:
do you want to leave it as `git@github.com:NervanaSystems/nnp-transformer.git` thought you had that for CI help

Reply:
I remember it's https, @DawnStone could you verify this?
In both cases, we could set `cmake -DCUSTOM_NNP_TRANSFORMER_GIT_REPOSITORY`

Reply:
For Jenkins, we needed to checkout the argon-transformer repo before running cmake, and reference the directory using `-DCUSTOM_NNP_TRANSFORMER_REPOSITORY`, although I am currently setting this as `-DCUSTOM_NNP_TRANSFORMER_DIR`.  I'll need to test this out to find out what it has really been doing during the build process.

Reply:
@DawnStone I guess you will need to check if you have  prefer  `https://github.com` over `git@github.com:` or vice versa

Reply:
@yxlao: I have verified that the Jenkins CI process does not use CUSTOM_NNP_TRANSFORMER_GIT_REPOSITORY, but successfully builds with the nnp-transformer repo using the CUSTOM_NNP_TRANSFORMER_DIR as the SOURCE_DIR.